src/main/resources/changesets/changelog-consent-82.0.xml

```diff
@@ -10,18 +11,14 @@
                 <constraints nullable="false"/>
             </column>
         </createTable>
-        <addPrimaryKey
-                columnNames="dataset_id, reference_id"
-                constraintName="join_table_key"
-                tableName="dar_dataset"/>
         <addForeignKeyConstraint baseColumnNames="dataset_id" baseTableName="dar_dataset" constraintName="fkDatasetId"
-                                 deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION"
+                                 deferrable="false" initiallyDeferred="false" onDelete="CASCADE" onUpdate="NO ACTION"
                                  referencedColumnNames="datasetid" referencedTableName="dataset"
-                                 referencesUniqueColumn="true"/>
+                                 referencesUniqueColumn="false"/>
         <addForeignKeyConstraint baseColumnNames="reference_id" baseTableName="dar_dataset" constraintName="fkReferenceId"
-                                 deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION"
+                                 deferrable="false" initiallyDeferred="false" onDelete="CASCADE" onUpdate="NO ACTION"
                                  referencedColumnNames="reference_id" referencedTableName="data_access_request"
-                                 referencesUniqueColumn="true"/>
+                                 referencesUniqueColumn="false"/>
```

Comment:
All schema changes you want to make need to be in a new, separate changelog file. This file has already been committed to prod so updating it here will have no effect.